themes/pom.xml

```diff
@@ -11,6 +11,108 @@
     <artifactId>keycloak-themes</artifactId>
     <name>Keycloak Themes</name>
     <description/>
+    <packaging>jar</packaging>
+
+    <properties>
+        <dir.common>src/main/resources/theme/keycloak/common/resources</dir.common>
+        <dir.account2>src/main/resources/theme/keycloak-preview/account/resources</dir.account2>
+        <args.npm.install>ci --no-optional --ignore-scripts</args.npm.install>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.8.0</version>
+                <executions>
+                    <!-- Download NPM tools -->
+                    <execution>
+                        <id>setup-node</id>
+                        <goals>
+                            <goal>install-node-and-npm</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                    </execution>
+                    <!-- Compile stuff -->
+                    <execution>
+                        <id>compile-account2</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <workingDirectory>${dir.account2}</workingDirectory>
+                            <arguments>run build --scripts-prepend-node-path</arguments>
+                        </configuration>
+                    </execution>
+                    <!-- Download NPM packages -->
+                    <execution>
+                        <id>npm-install-account2</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <configuration>
+                            <workingDirectory>${dir.account2}</workingDirectory>
+                            <arguments>${args.npm.install}</arguments>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <nodeVersion>v12.13.0</nodeVersion>
+                    <npmVersion>6.9.0</npmVersion>
+                    <installDirectory>node</installDirectory>
+                </configuration>
+            </plugin>
+        </plugins>
+
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <excludes>
+                    <exclude>**/node_modules/**/node_modules/**</exclude>
+                    <exclude>**/minimist/**</exclude>
+                    <exclude>**/mkdirp/**</exclude>
+                    <exclude>**/package.json</exclude>
```

Comment:
We should add `package-lock.json` as well. It's currently included in the distribution.